### PR TITLE
Fallout PnP 4.0

### PR DESCRIPTION
**Pull request by Skyeris**
## Changes / Comments

This is a complete overhaul of the Fallout PnP Character Sheet. It condenses everything into five tabs which are Prime, Skills, Combat, Inventory, and Vehicles. The Prime page contains your Character Information, Primary Stats, Secondary Stats, and Perks. Skills contains your Combat and Support skills with Temp boxes for bonuses you get from items. The Combat page now has Under Armor, Shields, Helmets, and easier method to add your weapons. The Inventory page has been made smaller for visibility. The Vehicles page is a new page that is specific for vehicles and it covers all necessary mechanics for them. The Vehicles page may be subject to changes since its extremely new.

This Character Sheet is meant to be used for the fan-made 4.0 version.

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [X] Does the pull request title have the sheet name(s)? Include each sheet name.
- [X] Is this a bug fix?
- [X] Does this add functional enhancements (new features or extending existing features) ?
- [X] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [X] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [X] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

**Comment by Anduh** [comment]
If the preview image is accurate, I have to say I'm extremely impressed over the looks considering the layout is done with `<table>` elements.

**Comment by Skyeris** [comment]
@Anduh We don't have <table> elements.

File: Fallout 4.0/sheet.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"html": "Fallout 4.0.html",
+	"css": "Fallout 4.0.css",
+	"authors": "Arnold Norona", "Osvel Cabrera",
```
Comment (Anduh) [inline Fallout 4.0/sheet.json]:
```suggestion
	"authors": "Arnold Norona, Osvel Cabrera",
```

File: Fallout 4.0/sheet.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	"roll20userid": "2370052", "761797",
```
Comment (Anduh) [inline Fallout 4.0/sheet.json]:
```suggestion
	"roll20userid": "2370052,  761797",
```

**Comment by Skyeris** [comment]
@Anduh Fixed it.

**Pull request merged**